src/backend/distributed/deparser/deparse_role_stmts.c

```diff
@@ -486,7 +486,6 @@ AppendGrantRoleStmt(StringInfo buf, GrantRoleStmt *stmt)
 	appendStringInfo(buf, "%s ", stmt->is_grant ? " TO " : " FROM ");
 	AppendRoleList(buf, stmt->grantee_roles);
 	AppendGrantWithAdminOption(buf, stmt);
-	AppendGrantedByInGrantForRoleSpec(buf, stmt->grantor, stmt->is_grant);
```

Comment:
Why do we need to remove this?

Reply:
There are multiple AppendGrantedByInGrantForRoleSpec added there which causes error